tests/test_geoprocessing.py

```diff
@@ -1338,7 +1338,8 @@ def test_zonal_statistics_nodata_is_zero(self):
         raster_path = os.path.join(self.workspace_dir, 'small_raster.tif')
         _array_to_raster(
             numpy.array([[1, 0], [1, 0]], dtype=numpy.int32), 0, raster_path,
-            origin=(origin_x, origin_y), pixel_size=(1.0, -1.0))
+            origin=(origin_x, origin_y), pixel_size=(1.0, -1.0),
+            projection_epsg=4326)

```

Comment:
This was a bug in the test: without specifying `projection_epsg`, it was defaulting to a different projection and did not actually overlap with the vector.